### DISCUSSION TITLE
Rating with default rating as empty

### DISCUSSION THREAD

**Discussion by ayan-akd** [saadeghi/daisyui#2269] [Q&A] [answered]
Like the title is there a way to get an empty rating at start by default?
I mean it's going to be not rated at the beginning.

**Comment by saadeghi** [comment, marked as answer]
In a form, using radio inputs, you need to send a data anyway so if it can be empty you would need a radio input for empty answer (0 stars)

Here you can see an example with `rating-hidden`:  
https://daisyui.com/components/rating/#with-rating-hidden  
The first radio input is hidden but is selectable if someone clicks the very start of the stars. This is a common UX and it allows users to erase their rating to zero. If you check that hidden input, it will be empty by default.

Example: https://play.tailwindcss.com/5kEHKTRdGS
